### DISCUSSION TITLE
《流浪地球》影评

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#13] [Announcements]
# 《流浪地球》影评

温馨提示：内容可能涉及剧透，如果你计划自己观看，就请不要阅读我的观后感。

https://zexwoo.blog/posts/gossip/the-wandering-earth-review/

**Comment by zj243** [comment]
写的很好，人类股骨的演讲比第一部赵今麦的演讲好多了 再加上今天看到纽约时报给的评分，在我这已经9分了！顺便博主可以评下三体和狂飙 都已经过半了

**Reply by ZexWoo** [reply]
《狂飙》我没看，《三体》可以写一下。
